### PR TITLE
Create new test module that uses the LLAPI 

### PR DESCRIPTION
**Pull request by alonre24**
Added a new flow test that loads this module and tests the LLAPI (currently contains a basic check only).
The module build is done by its own make file that is triggered in tests.sh before running flow tests.

### CODE REVIEW

**Comment by rafie** [comment]
Note: we should not be modifying `redismodule.h`.

**Comment by alonre24** [comment]
> Note: we should not be modifying `redismodule.h`.

File: tests/flow/tests_llapi.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import redis
+
+from includes import *
+import os
+
+'''
+python -m RLTest --test tests_llapi.py --module path/to/redisai.so
+'''
+
+#TEST_MODULE_PATH = "/home/alon/CLionProjects/RedisAI/bin/linux-x64-release/install-cpu/tests/redisai_testmodule/redisai_testmodule.so"
```
Comment (DvirDukhan) [inline tests/flow/tests_llapi.py:10]: remove remark

File: tests/module/LLAPI.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+	if(RedisAI_Initialize(ctx) != REDISMODULE_OK)
+		RedisModule_Log(ctx, "warning",
+		  "could not initialize RedisAI api, running without AI support.");
+
+	if(RedisModule_CreateCommand(ctx, "RAI_llapi.basic_check", RAI_llapi_basic_check, "",
+	  0, 0, 0) == REDISMODULE_ERR)
+		return REDISMODULE_ERR;
+	return REDISMODULE_OK;
+}
```
Comment (DvirDukhan) [inline tests/module/LLAPI.c:37]: empty line

**Review by DvirDukhan — APPROVED** [review]

**Review by DvirDukhan — APPROVED** [review]

**Pull request merged**